### PR TITLE
second task

### PR DESCRIPTION
**Pull request by monuelDePari**

### CODE REVIEW

File: Task2_decorator.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+@make_Quot
+@make_GT
+@make_IT
+@make_Amp
+def escape():
+    return string
+
+
+print(escape())
```
Comment (polyakovartem) [inline Task2_decorator.py]: https://docs.python.org/3/library/__main__.html

File: Task2_decorator.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Task2
+string = input()
+
+
+def make_GT(function) -> str:
```
Comment (polyakovartem) [inline Task2_decorator.py:5]: PEP8 function name violation

File: Task2_decorator.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        func = function()
+        make_replace = func.replace('"', '&quot')
+        return make_replace
+    return escapist
+
+
+@make_Quot
+@make_GT
+@make_IT
+@make_Amp
```
Comment (polyakovartem) [inline Task2_decorator.py]: have no clue what does GT, IT, Amp mean. Please use more the informative names

**Review by polyakovartem — CHANGES_REQUESTED** [review]
please remove copy-paste from the code

**Comment by monuelDePari** [comment]
> please remove copy-paste from the code

**Pull request closed without merging**